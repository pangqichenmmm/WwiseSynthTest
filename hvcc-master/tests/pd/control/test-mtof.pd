#N canvas -1433 -248 1219 770 10;
#X obj 407 579 + 1;
#X obj 25 57 loadbang;
#X obj 25 90 t b b b b b b b b b b b b b b b b b b b b b b b b b b
b b b;
#X obj 376 118 t b b b b b b b b b b b b b b b b b b b b b b b b b
b b b b;
#X obj 489 204 t b;
#X obj 520 203 t b b b b b b b b b b b b b b b b b b b b b b b b b
b b b b;
#X obj 633 289 t b;
#X obj 667 291 t b b b b b b b b b b b b b b b b b b b b b b b b b
b b b b;
#X obj 780 377 t b;
#X obj 815 377 t b b b b b b b b b b b b b b b b b b b b b b b b b
b b b b;
#X obj 928 463 t b;
#X obj 349 651 print;
#X obj 349 617 mtof;
#X obj 349 580 f -12;
#X text 407 82 // Note(joe): many triggers is because i don't have
[until] right now;
#X connect 0 0 13 1;
#X connect 1 0 2 0;
#X connect 2 0 13 0;
#X connect 2 1 13 0;
#X connect 2 2 13 0;
#X connect 2 3 13 0;
#X connect 2 4 13 0;
#X connect 2 5 13 0;
#X connect 2 6 13 0;
#X connect 2 7 13 0;
#X connect 2 8 13 0;
#X connect 2 9 13 0;
#X connect 2 10 13 0;
#X connect 2 11 13 0;
#X connect 2 12 13 0;
#X connect 2 13 13 0;
#X connect 2 14 13 0;
#X connect 2 15 13 0;
#X connect 2 16 13 0;
#X connect 2 17 13 0;
#X connect 2 18 13 0;
#X connect 2 19 13 0;
#X connect 2 20 13 0;
#X connect 2 21 13 0;
#X connect 2 22 13 0;
#X connect 2 23 13 0;
#X connect 2 24 13 0;
#X connect 2 25 13 0;
#X connect 2 26 13 0;
#X connect 2 27 13 0;
#X connect 2 28 3 0;
#X connect 3 0 4 0;
#X connect 3 1 4 0;
#X connect 3 2 4 0;
#X connect 3 3 4 0;
#X connect 3 4 4 0;
#X connect 3 5 4 0;
#X connect 3 6 4 0;
#X connect 3 7 4 0;
#X connect 3 8 4 0;
#X connect 3 9 4 0;
#X connect 3 10 4 0;
#X connect 3 11 4 0;
#X connect 3 12 4 0;
#X connect 3 13 4 0;
#X connect 3 14 4 0;
#X connect 3 15 4 0;
#X connect 3 16 4 0;
#X connect 3 17 4 0;
#X connect 3 18 4 0;
#X connect 3 19 4 0;
#X connect 3 20 4 0;
#X connect 3 21 4 0;
#X connect 3 22 4 0;
#X connect 3 23 4 0;
#X connect 3 24 4 0;
#X connect 3 25 4 0;
#X connect 3 26 4 0;
#X connect 3 27 4 0;
#X connect 3 28 5 0;
#X connect 4 0 13 0;
#X connect 5 0 6 0;
#X connect 5 1 6 0;
#X connect 5 2 6 0;
#X connect 5 3 6 0;
#X connect 5 4 6 0;
#X connect 5 5 6 0;
#X connect 5 6 6 0;
#X connect 5 7 6 0;
#X connect 5 8 6 0;
#X connect 5 9 6 0;
#X connect 5 10 6 0;
#X connect 5 11 6 0;
#X connect 5 12 6 0;
#X connect 5 13 6 0;
#X connect 5 14 6 0;
#X connect 5 15 6 0;
#X connect 5 16 6 0;
#X connect 5 17 6 0;
#X connect 5 18 6 0;
#X connect 5 19 6 0;
#X connect 5 20 6 0;
#X connect 5 21 6 0;
#X connect 5 22 6 0;
#X connect 5 23 6 0;
#X connect 5 24 6 0;
#X connect 5 25 6 0;
#X connect 5 26 6 0;
#X connect 5 27 6 0;
#X connect 5 28 7 0;
#X connect 6 0 13 0;
#X connect 7 0 8 0;
#X connect 7 1 8 0;
#X connect 7 2 8 0;
#X connect 7 3 8 0;
#X connect 7 4 8 0;
#X connect 7 5 8 0;
#X connect 7 6 8 0;
#X connect 7 7 8 0;
#X connect 7 8 8 0;
#X connect 7 9 8 0;
#X connect 7 10 8 0;
#X connect 7 11 8 0;
#X connect 7 12 8 0;
#X connect 7 13 8 0;
#X connect 7 14 8 0;
#X connect 7 15 8 0;
#X connect 7 16 8 0;
#X connect 7 17 8 0;
#X connect 7 18 8 0;
#X connect 7 19 8 0;
#X connect 7 20 8 0;
#X connect 7 21 8 0;
#X connect 7 22 8 0;
#X connect 7 23 8 0;
#X connect 7 24 8 0;
#X connect 7 25 8 0;
#X connect 7 26 8 0;
#X connect 7 27 8 0;
#X connect 7 28 9 0;
#X connect 8 0 13 0;
#X connect 9 0 10 0;
#X connect 9 1 10 0;
#X connect 9 2 10 0;
#X connect 9 3 10 0;
#X connect 9 4 10 0;
#X connect 9 5 10 0;
#X connect 9 6 10 0;
#X connect 9 7 10 0;
#X connect 9 8 10 0;
#X connect 9 9 10 0;
#X connect 9 10 10 0;
#X connect 9 11 10 0;
#X connect 9 12 10 0;
#X connect 9 13 10 0;
#X connect 9 14 10 0;
#X connect 9 15 10 0;
#X connect 9 16 10 0;
#X connect 9 17 10 0;
#X connect 9 18 10 0;
#X connect 9 19 10 0;
#X connect 9 20 10 0;
#X connect 9 21 10 0;
#X connect 9 22 10 0;
#X connect 9 23 10 0;
#X connect 9 24 10 0;
#X connect 9 25 10 0;
#X connect 9 26 10 0;
#X connect 9 27 10 0;
#X connect 10 0 13 0;
#X connect 12 0 11 0;
#X connect 13 0 0 0;
#X connect 13 0 12 0;
