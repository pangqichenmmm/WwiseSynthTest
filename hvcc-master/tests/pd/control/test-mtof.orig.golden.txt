NOTE: This is the actual expected output of Pds [mtof] with double implementation
[@ 0.000] print: 4.0879
[@ 0.000] print: 4.33098
[@ 0.000] print: 4.58851
[@ 0.000] print: 4.86136
[@ 0.000] print: 5.15043
[@ 0.000] print: 5.45669
[@ 0.000] print: 5.78116
[@ 0.000] print: 6.12493
[@ 0.000] print: 6.48914
[@ 0.000] print: 6.875
[@ 0.000] print: 7.28381
[@ 0.000] print: 7.71693
[@ 0.000] print: 8.1758
[@ 0.000] print: 8.66196
[@ 0.000] print: 9.17702
[@ 0.000] print: 9.72272
[@ 0.000] print: 10.3009
[@ 0.000] print: 10.9134
[@ 0.000] print: 11.5623
[@ 0.000] print: 12.2499
[@ 0.000] print: 12.9783
[@ 0.000] print: 13.75
[@ 0.000] print: 14.5676
[@ 0.000] print: 15.4339
[@ 0.000] print: 16.3516
[@ 0.000] print: 17.3239
[@ 0.000] print: 18.354
[@ 0.000] print: 19.4454
[@ 0.000] print: 20.6017
[@ 0.000] print: 21.8268
[@ 0.000] print: 23.1247
[@ 0.000] print: 24.4997
[@ 0.000] print: 25.9565
[@ 0.000] print: 27.5
[@ 0.000] print: 29.1352
[@ 0.000] print: 30.8677
[@ 0.000] print: 32.7032
[@ 0.000] print: 34.6478
[@ 0.000] print: 36.7081
[@ 0.000] print: 38.8909
[@ 0.000] print: 41.2034
[@ 0.000] print: 43.6535
[@ 0.000] print: 46.2493
[@ 0.000] print: 48.9994
[@ 0.000] print: 51.9131
[@ 0.000] print: 55
[@ 0.000] print: 58.2705
[@ 0.000] print: 61.7354
[@ 0.000] print: 65.4064
[@ 0.000] print: 69.2957
[@ 0.000] print: 73.4162
[@ 0.000] print: 77.7817
[@ 0.000] print: 82.4069
[@ 0.000] print: 87.3071
[@ 0.000] print: 92.4986
[@ 0.000] print: 97.9989
[@ 0.000] print: 103.826
[@ 0.000] print: 110
[@ 0.000] print: 116.541
[@ 0.000] print: 123.471
[@ 0.000] print: 130.813
[@ 0.000] print: 138.591
[@ 0.000] print: 146.832
[@ 0.000] print: 155.563
[@ 0.000] print: 164.814
[@ 0.000] print: 174.614
[@ 0.000] print: 184.997
[@ 0.000] print: 195.998
[@ 0.000] print: 207.652
[@ 0.000] print: 220
[@ 0.000] print: 233.082
[@ 0.000] print: 246.942
[@ 0.000] print: 261.626
[@ 0.000] print: 277.183
[@ 0.000] print: 293.665
[@ 0.000] print: 311.127
[@ 0.000] print: 329.628
[@ 0.000] print: 349.228
[@ 0.000] print: 369.994
[@ 0.000] print: 391.995
[@ 0.000] print: 415.305
[@ 0.000] print: 440
[@ 0.000] print: 466.164
[@ 0.000] print: 493.883
[@ 0.000] print: 523.251
[@ 0.000] print: 554.365
[@ 0.000] print: 587.33
[@ 0.000] print: 622.254
[@ 0.000] print: 659.255
[@ 0.000] print: 698.456
[@ 0.000] print: 739.989
[@ 0.000] print: 783.991
[@ 0.000] print: 830.609
[@ 0.000] print: 880
[@ 0.000] print: 932.328
[@ 0.000] print: 987.767
[@ 0.000] print: 1046.5
[@ 0.000] print: 1108.73
[@ 0.000] print: 1174.66
[@ 0.000] print: 1244.51
[@ 0.000] print: 1318.51
[@ 0.000] print: 1396.91
[@ 0.000] print: 1479.98
[@ 0.000] print: 1567.98
[@ 0.000] print: 1661.22
[@ 0.000] print: 1760
[@ 0.000] print: 1864.66
[@ 0.000] print: 1975.53
[@ 0.000] print: 2093
[@ 0.000] print: 2217.46
[@ 0.000] print: 2349.32
[@ 0.000] print: 2489.02
[@ 0.000] print: 2637.02
[@ 0.000] print: 2793.83
[@ 0.000] print: 2959.96
[@ 0.000] print: 3135.96
[@ 0.000] print: 3322.44
[@ 0.000] print: 3520
[@ 0.000] print: 3729.31
[@ 0.000] print: 3951.07
[@ 0.000] print: 4186.01
[@ 0.000] print: 4434.92
[@ 0.000] print: 4698.64
[@ 0.000] print: 4978.03
[@ 0.000] print: 5274.04
[@ 0.000] print: 5587.65
[@ 0.000] print: 5919.91
[@ 0.000] print: 6271.93
[@ 0.000] print: 6644.88
[@ 0.000] print: 7040
[@ 0.000] print: 7458.62
[@ 0.000] print: 7902.13
[@ 0.000] print: 8372.02
[@ 0.000] print: 8869.84
[@ 0.000] print: 9397.27
[@ 0.000] print: 9956.06
[@ 0.000] print: 10548.1
[@ 0.000] print: 11175.3
[@ 0.000] print: 11839.8
[@ 0.000] print: 12543.9
